doc/api/configuration.md

```diff
@@ -105,10 +105,12 @@ Automatically limit the size of the returned result sets (e.g., add a `LIMIT` at
 
 ## `displaylimit`
 
-Default: `None` (no limit)
+Default: `60`
 
 Automatically limit the number of rows displayed (full result set is still stored).
```

Comment:
Yes, since autolimit is adding the limit to the actual query

Will add to doc if the user has performance concern, then the `autolimit` is suggested